products/idn/docs/identity-now/saas-connectivity/connector-commands/account-disable.md

```diff
@@ -0,0 +1,73 @@
+---
+id: account-disable
+title: Account Disable
+pagination_label: Account Disable
+sidebar_label: Account Disable
+keywords: ['connectivity', 'connectors', 'account disable']
+description: Disable an account on the source.
+slug: /docs/saas-connectivity/commands/account-disable
+tags: ['Connectivity', 'Connector Command']
+---
+
+| Input/Output    |        Data Type        |
+| :-------------- | :---------------------: |
+| Input - Disable | StdAccountDisableInput  |
+| Output -Disable | StdAccountDisableOutput |
+
+### Example StdAccountDisableInput
+
+```javascript
+"key": {
+    "simple": {
+        "id": "john.doe"
+    }
+}
+```
+
+### Example StdAccountDisableOutput
+
+```javascript
+{
+    "identity": "john.doe",
+    "key": {
+        "simple": {
+            "id": "john.doe"
+        }
+    },
+    "disabled": false,
+    "locked": false,
+    "attributes": {
+        "id": "john.doe",
+        "displayName": "John Doe",
+        "email": "example@sailpoint.com",
+        "entitlements": [
+            "administrator",
+            "sailpoint"
+        ]
+    }
+}
+```
+
+## Description
+
+You typically invoke the account disable command during the joiner, mover, leaver (JML) lifecycle. An identity’s leaving from the organization or change to a role that does not require access to one or more accounts triggers the account disable command.
+
+Disabling accounts is generally preferred if the source supports account disabling so the account data remains for later reactivation or inspection. If the source does not support account disabling or deleting accounts is preferred when an identity leaves the organization, the connector performs the necessary steps to delete an account with the account disable function.
```

Comment:
Can you change "If the source does not support account disabling or deleting" to "If the source does not support account disabling, deleting..."